hive/src/test/java/com/netease/arctic/hive/catalog/TestArcticHiveCatalog.java

```diff
@@ -41,4 +56,61 @@ public static Object[] parameters() {
     return new Object[] {new HiveCatalogTestHelper(TableFormat.MIXED_HIVE, TEST_HMS.getHiveConf()),
                          new HiveCatalogTestHelper(TableFormat.ICEBERG, TEST_HMS.getHiveConf())};
   }
+
+  @Test
+  public void testArcticTableRootLocation() throws TException {
+    ArcticCatalog hiveCatalog = getCatalog();
+    // iceberg table doesn't support to set hive's property
+    if (hiveCatalog instanceof BasicIcebergCatalog) {
+      return;
+    }
+
+    String dbName = "arctic_location_test";
+    String unkeyedTable = "unkeyed_test_hive_table_tmp";
+    String keyedTable = "keyed_test_hive_table_tmp";
+    boolean createDatabase = false;
+    createDatabase = !(hiveCatalog.listDatabases().contains(dbName));
+    if (createDatabase) {
+      hiveCatalog.createDatabase(dbName);
+    }
+
+    final TableIdentifier unKeyedHiveTableIdentifier =
+            TableIdentifier.of(TableTestHelper.TEST_CATALOG_NAME, dbName, unkeyedTable);
+
+    final TableIdentifier keyedHiveTableIdentifier =
+            TableIdentifier.of(TableTestHelper.TEST_CATALOG_NAME, dbName, keyedTable);
+
+    hiveCatalog.newTableBuilder(unKeyedHiveTableIdentifier, HIVE_TABLE_SCHEMA)
+            .withPartitionSpec(HIVE_SPEC)
+            .create().asUnkeyedTable();
+
+    Map<String,String> tableParameter =  TEST_HMS.getHiveClient()
+            .getTable(dbName, unkeyedTable).getParameters();
+    Assert.assertTrue(tableParameter.containsKey(ARCTIC_TABLE_ROOT_LOCATION));
```

Comment:
Abstract the validating codes into a private method to reduce code redundancy.

Reply:
done